changelogs/unreleased/8228-database_pool_report.yml

```diff
@@ -0,0 +1,6 @@
+description: Update database pool status reporting for the new scheduler
+issue-nr: 8228
+change-type: minor
+destination-branches: [master]
+sections:
+  minor-improvement: "{{description}}"
```

Comment:
Does this deserve a change entry, considering that the new scheduler has not been publicly released? Or has it?

Reply:
It is a user visible change, but perhaps I should add more detail?

Reply:
I mean the entire scheduler is new, so I would assume that to be one change entry rather than a separate one for each prt it affects.

Reply:
I don't want to lose the details, but we can probably collect them at the end